.github/dependabot.yml

```
@@ -0,0 +1,9 @@
+version: 2
+updates:
+- package-ecosystem: maven
+  directory: "/"
```

```suggestion
  directory: "/"
  vendor: true
```

We vendor our dependencies in the `vendor/` directory.